src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Python/Bindings.cpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+#include <pybind11/pybind11.h>
+#include <type_traits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/TimeDerivOfShift.hpp"
+#include "Utilities/ErrorHandling/SegfaultHandler.hpp"
+
+namespace py = pybind11;
+
+namespace GeneralizedHarmonic::py_bindings {
+
+namespace {
+template <size_t Dim>
+void bind_impl(py::module& m) {  // NOLINT
+  m.def(
+      "time_deriv_of_shift",
+      static_cast<tnsr::I<DataVector, Dim> (*)(
+          const Scalar<DataVector>&, const tnsr::I<DataVector, Dim>&,
+          const tnsr::II<DataVector, Dim>&, const tnsr::A<DataVector, Dim>&,
+          const tnsr::iaa<DataVector, Dim>&, const tnsr::aa<DataVector, Dim>&)>(
+          &::gh::time_deriv_of_shift),
+      py::arg("lapse"), py::arg("shift"), py::arg("inverse_spatial_metric"),
+      py::arg("spacetime_unit_normal"), py::arg("phi"), py::arg("pi"));
+}
+}  // namespace
+
+PYBIND11_MODULE(_PyGeneralizedHarmonic, m) {  // NOLINT
```

Comment:
Please rebase on the latest `develop` branch and rename this `_PyGeneralizedHarmonic` -> `_Pybindings`. That was a change made in #5058.